### PR TITLE
Store variables, and get functions

### PR DESCRIPTION
**Pull request by charlesstrange2**
payment page store observers
payment page wiring to observers

### CODE REVIEW

**Review by gingerbreadpup — APPROVED** [review]
:icecream: 
:shipit:

File: src/store.js [hunk 1]
```diff
@@ -2,12 +2,25 @@ import NetMgr from './netMgr.js';
...
     vouchers: [],
     recVouchers: [],
     netMgr : NetMgr
 };
 
+store.getVoucherPaymentState = function() {
+
+    this.netMgr.apiGet('traders/'+this.user.id +'/vouchers/history' , function(response) {
+        this.trader.pendedVouchers.splice(0,this.trader.pendedVouchers.length, response.data);
+        console.log(this.trader.pendedVouchers);
```
Comment (katjam) [inline src/store.js]: obvs not keeping this?

File: fixtures/fixtures.js [hunk 2]
```diff
@@ -10,32 +10,32 @@ var Fixtures = {
     traderVoucherHistory: {
```
Comment (katjam) [inline fixtures/fixtures.js:10]: I will build the api views based on these fixtures.

**Comment by charlesstrange2** [comment]
yup. that one might be... tricky. :-(

File: src/store.js (hunk 1 above)
Comment (charlesstrange2) [reply src/store.js]: er, no. will change

**Pull request merged**